sdk/js/src/examples/simpleSearcherLimo.ts

```diff
@@ -186,6 +192,10 @@ class SimpleSearcherLimo {
     this.recentBlockhash[update.chain_id] = update.blockhash;
   }
 
+  async removeOpportunitiesHandler(opportunityDelete: OpportunityDelete) {
+    console.log("Removing opportunities", opportunityDelete);
```

Comment:
nit: Maybe add a comment that devs should implement their own removal logic or change the log message to something like `Opportunity X doesn't exist anymore`. Right now it feels like something is happening in the client but it's not